Libplanet.Action/ActionEvaluator.cs

```diff
@@ -134,7 +125,7 @@ public class ActionEvaluator : IActionEvaluator
                 }
 
                 var evaluations = ImmutableList<ActionEvaluation>.Empty;
-                if (_policyBeginBlockActionsGetter(block) is { } beginBlockActions &&
+                if (_policyActionsGetterCollection.BeginBlockActionsGetter(block) is { } beginBlockActions &&
```

Comment:
It seems too long... 🤔

Reply:
How about shortening the name?
For example  
`BeginBlockActionsGetter` -> `BeginActions`
`_policyActionsGetterCollection` -> `_policyActions`

Reply:
To clarify meanings of each variable, I decided to insert newline instead of shortening variable names.